### DISCUSSION TITLE
Project readme should provide information on how to setup an environment suitable for "I tried to reproduce the issue when..."

### DISCUSSION THREAD

**Discussion by BawdyInkSlinger** [uBlockOrigin/uAssets#19500] [Ideas]
When you try to create an issue here, it has a few checkboxes:

![image](https://github.com/uBlockOrigin/uAssets/assets/51732963/70554e4a-9b53-4564-8524-dfaa241a672e)

I've noticed the contributors time and again have to bring this up to people in the comments. I think these steps would be followed by more people if it was easier to learn how to do them. One way to make it easier is to include the information in the readme, either as external links or inlined documentation. 

For example, I use Firefox and I know how to disable extensions. But I only know how to do it by toggling them off, one by one. I'm not sure how to create a new FF profile. For all I know, there is a way of starting the browser in "Safe Mode" or "Extension Troubleshooting Mode," which provides the perfect environment for reporting issues. 

I envision new sections for each major browser, explaining how to set it up to report an issue and then revert back. Any thoughts? I'm not necessarily saying, "Somebody should do this." I'm saying, "Should this be done? If so, I'd be willing to help."

---

**I know googling this would enlighten me, that's not the point. The point is the time and effort it takes to find what I'm looking for and the time and effort of putting my environment back the way it was.**

**Comment by MasterKia** [comment]
I think the GitHub issue template should match the "Report a website" template from uBO.
